components/content-service/pkg/service/blob-service.go

```diff
@@ -22,6 +22,8 @@ import (
 type BlobService struct {
 	cfg storage.Config
 	s   storage.PresignedAccess
+
+	api.UnimplementedBlobServiceServer
```

Comment:
I understand this change is necessary due to some deliberately non-backwards compatible, idiosyncratic design decisions (https://github.com/grpc/grpc-go/issues/3669#issuecomment-695022942).

I'm am however not convinced the unimplemented default implementation is preferable to things breaking at compile time. If someone modifies the interface, they should be made aware of the consequences of that action as early as possible.

We can't disable this behaviour though, so we'll have to live with it.